pkg/loghttp/push/push_test.go

```diff
@@ -196,6 +204,29 @@ func TestParseRequest(t *testing.T) {
 			expectedBytes:                   len("fizzbuzz") + 2*len("a") + 2*len("b"),
 			expectedLines:                   1,
 			expectedBytesUsageTracker:       map[string]float64{`{foo="bar2"}`: float64(len("fizzbuzz") + 2*len("a") + 2*len("b"))},
+			expectedLabels:                  labels.FromStrings("foo", "bar2"),
+		},
+		{
+			path:                      `/loki/api/v1/push`,
+			body:                      `{"streams": [{ "stream": { "foo": "bar2", "job": "stuff" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`,
+			contentType:               `application/json`,
+			valid:                     true,
+			enableServiceDiscovery:    true,
+			expectedBytes:             len("fizzbuzz"),
+			expectedLines:             1,
+			expectedBytesUsageTracker: map[string]float64{`{foo="bar2", job="stuff"}`: float64(len("fizzbuss"))},
+			expectedLabels:            labels.FromStrings("foo", "bar2", LabelServiceName, "stuff"),
+		},
+		{
+			path:                      `/loki/api/v1/push`,
+			body:                      `{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`,
+			contentType:               `application/json`,
+			valid:                     true,
+			enableServiceDiscovery:    true,
+			expectedBytes:             len("fizzbuzz"),
+			expectedLines:             1,
+			expectedBytesUsageTracker: map[string]float64{`{foo="bar2"}`: float64(len("fizzbuss"))},
+			expectedLabels:            labels.FromStrings("foo", "bar2", LabelServiceName, ServiceUnknown),
```

Comment:
`expectedLabels` I can't find its usage in the test 🤔

Reply:
Good catch. I _really_ thought I added it here but I must not have because one of the tests was wrong.